test/org/sagebionetworks/bridge/services/ExternalIdServiceTest.java

```diff
@@ -156,4 +148,150 @@ public void deleteExternalIdsWithValidationEnabled() {
         }
         verifyNoMoreInteractions(externalIdDao);
     }
+    
+    @Test
+    public void createExternalIdValidatedWithValue() {
+        setupExternalIdTest(true, null);
+        
+        externalIdService.reserveExternalId(STUDY, EXT_ID, HEALTH_CODE);
+        externalIdService.assignExternalId(STUDY, EXT_ID, HEALTH_CODE);
+        
+        // Adding validated ID, reserve it
+        verifySetAsReservation(EXT_ID);
+    }
+
+    @Test
+    public void createExternalIdNotValidatedWithValue() {
+        setupExternalIdTest(false, null);
+        
+        externalIdService.reserveExternalId(STUDY, EXT_ID, HEALTH_CODE);
+        externalIdService.assignExternalId(STUDY, EXT_ID, HEALTH_CODE);
+        
+        // Adding unvalidated ID, set as an option
+        verifySetAsOption(EXT_ID);
+    }
```

Comment:
Okay, I've removed all but one test for not validated path. This does depend on knowing these service methods do nothing if that flag is set to false, were that to change, the tests might not catch change in behavior.